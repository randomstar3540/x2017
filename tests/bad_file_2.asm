FUNC LABEL 0
    MOV REG 0 VAL 8
    PRINT REG 0
    RET
FUNC LABEL 0
    MOV REG 0 VAL 8
    PRINT REG 0
    RET
