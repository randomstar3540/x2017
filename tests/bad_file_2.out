Not an x2017 formatted file
